scripts/monorepo_fixup.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+# This script should be run when updating a developer's working copy to the monorepo codebase
+
+rm -rf node_modules build packages/*/node_modules packages/*/dist packages/*/lib
+yarn
+yarn modules
```

Comment:
@vigneshm do we still need `yarn fixup` or `yarn cleanModules`?

Reply:
This sounds good to me. They aren't used as often and I'm fine with moving it into a script.